Userland/Libraries/LibCore/System.cpp

```diff
@@ -81,10 +81,33 @@ ErrorOr<void> pledge(StringView promises, StringView execpromises)
     HANDLE_SYSCALL_RETURN_VALUE("pledge", rc, {});
 }
 
+static bool dynamic_loader_unveiled { false };
+
+static ErrorOr<void> unveil_dynamic_loader()
+{
+    if (dynamic_loader_unveiled)
+        return {};
+    // FIXME: Try to find a way to not hardcode the dynamic loader path.
+    auto const dynamic_loader_path = "/usr/lib/Loader.so"sv;
+    auto const dynamic_loader_permissions = "x"sv;
```

Comment:
These should probably be `constexpr`/`consteval`, although I'm not sure if it makes any difference in practice.